serverAkkaHttp/src/main/scala/RPCRouter.scala

```diff
@@ -88,10 +95,8 @@ trait Router extends RPCServer with PathMacro with MetaDataMacro {
   def commandArgs(request: Json, token: Option[String]): Map[String, Json] =
     request.as[Map[String, Json]].right.get.withToken(token)
 
-  def queryArgs(params: Map[String, String], token: Option[String]): Map[String, Json] = {
-    import io.circe.syntax._
-    params.mapValues(_.asJson).withToken(token)
-  }
+  def queryArgs(params: Map[String, String], token: Option[String]): Either[Exception, Map[String, Json]] =
+    params.mapValues(parse(_)).sequenceU.right.map(_.withToken(token))
```

Comment:
if you import `import cats.syntax.either._` you can treat Either as right-biased (in Scala 2.12 it will just work without the implicit conversion)